napari/layers/image/_image_constants.py

```diff
@@ -29,6 +29,23 @@ class Interpolation(StringEnum):
     SPLINE36 = auto()
 
 
+InterpolationSubset = StringEnum(
+    'InterpolationSubset',
+    [
+        (a.name, a.value)
+        for a in Interpolation
+        if a
+        in [
+            Interpolation.BICUBIC,
+            Interpolation.BILINEAR,
+            Interpolation.KAISER,
+            Interpolation.NEAREST,
+            Interpolation.SPLINE36,
+        ]
+    ],
+)
+
+
```

Comment:
I'd be more inclined to do this if we were using it for something other than just the GUI... but this also makes it less useful as a "subset"

```python
In [5]: from napari.layers.image._image_constants import Interpolation, InterpolationSubset

In [6]: Interpolation.BILINEAR is InterpolationSubset.BILINEAR
Out[6]: False

In [7]: Interpolation.BILINEAR == InterpolationSubset.BILINEAR
Out[7]: False
```

```suggestion
```